rules/helm/package.sh

```diff
@@ -4,7 +4,7 @@ set -o errexit -o nounset
 
 build_dir="tmp/build/${PACKAGE_DIR}"
 mkdir -p "${build_dir}"
-cp -L --recursive "${PACKAGE_DIR}"/* "${build_dir}"
+rsync -a "${PACKAGE_DIR}"/ "${build_dir}"/
```

Comment:
rsync messes with the final .tgz output. I confirmed that OS X dislikes `--recursive` as much as it dislikes `--dereference`. So, could you do a `cp -L -R "${PACKAGE_DIR}"/* "${build_dir}"` instead? It will work cross-platform.